src/deepl.ts

```diff
@@ -1,11 +1,14 @@
 import { default as axios, AxiosInstance } from "axios";
 import qs from "qs";
 import { Logger } from "@slack/logger";
+import {AssertionError} from "assert";
```

Comment:
You can use code formatter

Reply:
Actually I fogot to remove this line. Will do. Thanks